sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/ContinuousExecution.scala

```diff
@@ -117,6 +117,7 @@ class ContinuousExecution(
     // For at least once, we can just ignore those reports and risk duplicates.
     commitLog.getLatest() match {
       case Some((latestEpochId, _)) =>
+        updateStatusMessage(s"Getting offsets from latest epoch $latestEpochId")
```

Comment:
nit: I'd mention that we're restarting the streaming query from an existing epoch in this message, to contrast with "starting new query" in the other branch

Reply:
Fixed.